exercises/etl/etl_test.rb

```diff
@@ -32,27 +41,67 @@ def test_more_keys
     assert_equal expected, ETL.transform(old)
   end
 
-  def test_full_dataset
+  def test_multiple_scores_with_differing_numbers_of_letters
     skip
     old = {
-      1 => %w(A E I O U L N R S T),
-      2 => %w(D G),
-      3 => %w(B C M P),
-      4 => %w(F H V W Y),
-      5 => %w(K),
-      8 => %w(J X),
-      10 => %w(Q Z)
```

Comment:
Unsure it was 'arbitrary' as it does save some typing... like 2/3rds for each single character "word", after the first element.  For a single element entry, it is identical in character length.  It was probably a convenience of typing, and something that is idiomatic (even encouraged in the style guide).

Consistency is good though.